### PR TITLE
error capture docs

### PR DESCRIPTION
**Pull request by dschep**
implementation PR: https://github.com/serverless/enterprise-plugin/pull/185

### CODE REVIEW

**Comment by skierkowski** [comment]
@dschep I made a few small tweaks/additions. Can you review those? Other than that 👍 .

File: docs/error-capture.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  return {
+    statusCode: 500,
+    body: JSON.stringify({ name: 'bob' }),
+  };
+};
+```
+
+And to import it instead, import with `const { captureError } = require('./serverless-sdk')` then
+call `captureError` instead of `context.captureError`.
```
Comment (alexdebrie) [inline docs/error-capture.md]: I think it's worth being explicit here and showing a full example as well. Something like:

```javascript

// getUser.js

const { captureError } = require('./serverless-sdk')

module.exports.getUser = (id) => {
  try {
    const user = await getUserById(id)
    return user
  } catch (error) {
    captureError(error)
    return
  }
}
```

Comment (dschep) [reply docs/error-capture.md]: done

File: docs/error-capture.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```javascript
+const { captureError } = require('./serverless_sdk')
```
Comment (dschep) [inline docs/error-capture.md]: switched to _ with merge of python sdk pr. in that pr switched to underscore to simplify and avoid branching on name for python & node (python doesn't support modules/packages with - in them)

**Pull request merged**